src/check/arbitrary/ObjectArbitrary.ts

```diff
@@ -397,7 +397,7 @@ function jsonSettings(stringArbitrary: Arbitrary<string>, constraints?: number |
   const values = [
     boolean(),
     maxSafeInteger(),
-    double({ next: true, noDefaultInfinity: true, noNaN: true }),
+    double({ noDefaultInfinity: true, noNaN: true }).filter((d) => !Object.is(d, -0)),
```

Comment:
Fix has to be done on `main` branch too. It should also be back-ported as the change (from `double()` to `double({next:true})`) introduced a breaking change.

Reply:
It does not seem that `-0` is an illegal value.

Nonetheless, there is a strange behaviour in JavaScript with negative zeros and `JSON.*` methods:
- `JSON.parse('{"a": -0}')` preserves `-0`
- `JSON.stringify({a: -0})` does not and produces `{"a":0}`

Reply:
See https://tools.ietf.org/html/rfc7159#page-6